src/parser/parse_tree_cl.c

```diff
@@ -2742,6 +2742,13 @@ parser_print_tree (PARSER_CONTEXT * parser, const PT_NODE * node)
 	      string = pt_append_nulstring (parser, string, user_text_buffer);
 	    }
 	}
+      if ((parser->custom_print & PT_PRINT_HOST_VAR_COUNT) != 0)
+	{
+	  char host_var_count[12];
+	  snprintf (host_var_count, sizeof (host_var_count), "%d", parser->host_var_count + parser->auto_param_count);
+	  string = pt_append_nulstring (parser, string, ";host_var=");
```

Comment:
개수를 의미하기 때문에 s를 붙이는게 좋겠다는 생각입니다. 검토 한 번 부탁드려요.
```suggestion
	  string = pt_append_nulstring (parser, string, ";host_vars=");
```

Reply:
bind parameter count? 라고 하는 것은 어떨까요?

Reply:
host_var_count? host_var_cnt?

Reply:
host와 auto var을 모두 포함하므로 다른 이름이 필요할 듯 합니다.